src/js/intlTelInput.js.ejs

```diff
@@ -246,6 +250,11 @@ Plugin.prototype = {
       // this is useful in lots of places
       this.countryListItems = this.countryList.children(".country");
     }
+    
+    // add the RTL class if requested
+    if(this.options.rtl === true) {
+    	this.telInputRootElement.addClass("iti-rtl");
```

Comment:
no need to have this cached if we're only going to use it once - just get the parent here instead